docs/site/Components.md

```diff
@@ -0,0 +1,29 @@
+---
+lang: en
+title: 'Components'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Components.html
+---
+
+## Overview
+
+A `Component` is an extensibility class, which allows for creating various types
+of LoopBack 4 extensions.
+
+Apart from its own properties, `Component` class can have the following
+properties:
+
+- `controllers` - An array of [controller](Controllers.md) classes.
+- `providers` - A map of providers to be bound to the application context.
+- `classes` - A map of TypeScipt classes to be bound to the application context.
+- `servers` - A map of name/class pairs for [servers](Server.md).
+- `lifeCycleObservers` - An array of [life cycle observers](Life-cycle.md).
+- `bindings` - An array of [bindings](Bindings.md) to be aded to the application
+  context.
```

Comment:
AFAICT, there is a list of thing a component can contribute in [Using components](https://loopback.io/doc/en/lb4/Using-components.html) too. I am concerned that it's too difficult to keep these two lists in sync. Please take a look at both pages and decide where to put a single instance of this list, and remove it from the other place.

Reply:
"Using components" is just limited to how to use components now. Other details have been moved to `docs/site/Components.md`.